### ISSUE TITLE
cannot reproduce the results (tiny version)

### ISSUE THREAD

**Issue by JudasDie** [zhiqi-li/BEVFormer#81]
Hi, thanks for the great work. 
I reproduce the tiny version with batchsize=4, buy only get 0.318 NDS on val, which is far away from the reported 0.35. Any suggestion?

Thanks in advance!

**Comment by zhiqi-li** [comment]
With default settings, can you reproduce our  results? When you set the batchsize=4, what learning rate did you use?

**Comment by JudasDie** [comment]
> With default settings, can you reproduce our results? When you set the batchsize=4, what learning rate did you use?

I didn't change the learning rate since Base version works well with a larger batch size. I'm re-training the model with default setting, but it may take a while.

**Comment by zhiqi-li** [comment]
Hi, when you adjust the total batch size, you should adjust the learning rate accordingly. You can refer to 如何选择模型训练的batch size和learning rate - 科技猛兽的文章 - 知乎
https://zhuanlan.zhihu.com/p/363645881

**Comment by JudasDie** [comment]
> Hi, when you adjust the total batch size, you should adjust the learning rate accordingly. You can refer to 如何选择模型训练的batch size和learning rate - 科技猛兽的文章 - 知乎 https://zhuanlan.zhihu.com/p/363645881

Thanks, I will try to reproduce the model with batch=1. The results will be posted here later.

**Comment by JudasDie** [comment]
> Hi, when you adjust the total batch size, you should adjust the learning rate accordingly. You can refer to 如何选择模型训练的batch size和learning rate - 科技猛兽的文章 - 知乎 https://zhuanlan.zhihu.com/p/363645881

I can reproduce the results with default settings. But it doesn't work with larger batch size even I increase the learning rate.

**Comment by zhiqi-li** [comment]
With batchsize=2, the bevformer-tiny works well in my experiments and I provided related pre-trained models and config.
